libs/eo/origin-of-energy/shell/src/lib/eo-origin-of-energy.service.ts

```diff
@@ -23,22 +23,22 @@ import {
 import { map, Observable } from 'rxjs';
 
 export interface EoOriginOfEnergyResponse {
-  energysources: [
+  energySources: [
```

Comment:
Why is this camelCase and not the entries?
Like naturalgas etc.

Reply:
Because we are not in control of what comes back inside the ratios object, but everything outside it is ours to control, and there we use camelCase :)

I asked Claus to use a toLowerCase() function on the entries in ratios, because otherwise they would have returned "Straw" instead of "straw", and NaturalGas instead of "naturalGas". And since i dont want him to make a lot of if statements, we opted for toLowerCase :)